documentation/assemblies/assembly-managing-users-oc.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+//master.adoc
+:parent-context: {context}
+:context: managing-users-oc
+:cmdcli: oc
+
+[id='managing-users-{context}']
+= Managing users on OpenShift
+
+{ProductName} user management is only supported when using the `standard` authentication service. On OpenShift, users can
+be managed using the OpenShift command-line tools.
+
+.Prerequisites
+
+* You must have already created an xref:con-address-space-messaging[address space]
+
+include::../modules/proc-creating-users-cli.adoc[leveloffset=+1]
+include::../modules/proc-deleting-users-cli.adoc[leveloffset=+1]
+include::../modules/proc-managing-user-permissions-cli.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:enmasse_rest_api[]
```

Comment:
Period

Reply:
I'm assuming you meant the period needed on the Prereq; done